.github/workflows/build-and-release-stable.yml

```diff
@@ -59,9 +59,15 @@ jobs:
         with:
           version: latest
           args: release --rm-dist --skip-validate
+      - name: bump homebrew-core formula
+        uses: mislav/bump-homebrew-formula-action@v2
+        with:
+          formula-name: mesheryctl
+          download-url: https://github.com/meshery/meshery.git
```

Comment:
Update need to specify new location and release tag.

Reply:
I'm sorry didn't understand your question. Are you asking if we need to explicitly configure new tap location and release tag? 

If yes, this gh action assumes the location to be `homebrew-core` if not specified, tag defaults to the currently pushed tag if not specified.